src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -515,16 +515,16 @@ public int Space_WithinSquares
             set { SetBooleanOption(CSharpFormattingOptions.SpaceWithinSquareBrackets, value); }
         }
 
-        public int Style_PreferIntrinsicPredefinedTypeKeywordInDeclaration
+        public string Style_PreferIntrinsicPredefinedTypeKeywordInDeclaration
```

Comment:
The automation object is used for a few things:

1. Programmatic access through DTE.
2. Saving and loading settings from .vssettings files
3. Applying profiles.

@balajikris, is this value new for Visual Studio "15" or did we ship it in Visual Studio 2015 at all? If we already shipped it then yes, this is a breaking change and we might need to do something else. If this is new for "15", then we're probably OK letting this slide. However, we do need to be aware that we might see the old int values now being passed as a stringly typed...something for people upgrading from earlier "15" builds, or people who already created team .vssettings files, etc. It might be worth ensuring we at least don't do something bad in that scenario.